### PR TITLE
Update Quiz Page

### LINKED ISSUE #1052: Update the /quiz UI
### What's your idea?

The `acmcsuf.com/quiz` page is slightly out of date due to the new teams that we have. We need to update the data along with doing a redesign to fit with the style of the rest of the website.
quiz

### PR DESCRIPTION
**Pull request by 26samaahmed**
Fix #1052 
Screenshot 2024-04-09 at 10 56 43 PM

### CODE REVIEW

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
Some nitpicks:

### Color of button on results page looks weird in dark mode

image

### Gaps between team tiles could be increased horizontally and vertically

image

### Update results title to include "Quiz" e.g. "Quiz Results"

### Add 10% to Open Source Software team results

File: src/routes/(site)/quiz/quiz.svelte [hunk 1]
```diff
@@ -88,6 +88,7 @@
   <!-- DISPLAY THE QUIZ QUESTIONS -->
   {#if !showResults}
     <h1>ACM TEAM QUIZ</h1>
+    <p><i>What team are you a part of?</i></p>
```
Comment (JustinStitt) [inline src/routes/(site)/quiz/quiz.svelte]: nit: s/What/Which

I think that reads a bit better.

**Comment by JustinStitt** [comment]
Your PR description says "Fix https://github.com/EthanThatOneKid/acmcsuf.com/issues/1052" but https://github.com/EthanThatOneKid/acmcsuf.com/issues/1052 mentions updating more than just the UI: `We need to update the data along with doing a redesign to fit with the style of the rest of the website.`

I think a bit more detail in your PR descriptions can help reviewers understand what is going on. With that being said, and with Ethan's review, you're on the right track 👍 🦖

**Comment by 26samaahmed** [comment]
I have fixed the buttons on the results page as well as increase the gap between the team tile. I also noticed that the progress bar for some teams disappears on mobile so I increased the padding for the team tiles to be responsive.

Screenshot 2024-04-16 at 2 22 33 PM  Screenshot 2024-04-16 at 2 30 30 PM


Screenshot 2024-04-16 at 2 21 38 PM

Screenshot 2024-04-16 at 2 23 56 PM

**Comment by k4rni** [comment]
## TODO
- [x] Choice boxes should be a set size 
image
- [x] Change the color of the choice boxes to be a darker color so the words pop out more (maybe it could be `var(--acm-blue)` while the buttons are `var(--acm-bluer)`
- [x] Remove glow effect and replace with a solid border on hover
image
- [x] Remove the general team, marketing team, special events team, and nodebuds team in the final results
image

**Comment by 26samaahmed** [comment]
For removing general, marketing, nodebuds, and special events team tiles from quiz results, the only way I was able to remove them is by deleting them from the teams.json since that's where we're getting the data from. Is there another way to remove them without affecting the rest of the website or modifying the teams.json file? @karnikaavelumani @EthanThatOneKid

**Comment by 26samaahmed** [comment]
I removed the glow effect on hover and replaced it with a border color. I also changed the background color to be more readable and made the height fixed.

Screenshot 2024-04-29 230433

Screenshot 2024-04-29 230523

Screenshot 2024-04-29 231948

File: src/routes/(site)/quiz/quiz.svelte [hunk 2]
```diff
@@ -55,16 +57,29 @@
...
+          tallies[match] = 1;
+        }
+      }
+      return tallies;
+    }, {} as Record<string, number>);
+  }
+
+  $: talliedResponses = tallyResponses(responses);
 
+  // Bug in this definition
```
Comment (EthanThatOneKid) [inline src/routes/(site)/quiz/quiz.svelte]: TODO(@EthanThatOneKid): Fix this lol.

File: requirements.txt
Comment (EthanThatOneKid) [inline requirements.txt:1]: Python moment

File: src/routes/(site)/quiz/quiz.svelte (hunk 2 above)
Comment (EthanThatOneKid) [reply src/routes/(site)/quiz/quiz.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1059/commits/fcbc9fe8ab4c23dae9939c151dfc9d496ea2edaa>.

**Review by k4rni — APPROVED** [review]
LGTM!

**Review by EthanThatOneKid — APPROVED** [review]

**Pull request merged**